src/framework/ui/view/interactiveprovider.cpp

```diff
@@ -282,6 +356,36 @@ RetVal<InteractiveProvider::OpenData> InteractiveProvider::openQml(const UriQuer
     return result;
 }
 
+RetVal<Val> InteractiveProvider::openStandardDialog(const QString& type, const QString& title, const framework::IInteractive::Text& text,
+                                                    const framework::IInteractive::ButtonDatas& buttons, int defBtn,
+                                                    const framework::IInteractive::Options& options)
+{
+    QmlLaunchData* data = new QmlLaunchData();
+    fillStandatdDialogData(data, type, title, text, buttons, defBtn, options);
+
+    emit fireOpenDialog(data);
```

Comment:
I believe that it is actually better to just build uri and open it like any other dialogue.
Like:
```
    UriQuery q("musescore://interactive/error");
    q.addParam("title", title);
    q.addParam("text", text.toJson());
    q.addParam("buttons", buttons.toJson());
    q.addParam("defBtn", defBtn);
    q.addParam("options", options.toJson());

```

Reply:
This is a desire, I do not insist :)